modules/300-prometheus/images/grafana-dashboard-provisioner/hooks/dashboard_provisioner.sh

```diff
@@ -31,12 +34,20 @@ function __config__() {
 EOF
 }
 
+function clear_tmp() {
+  find /tmp/dashboards/ -mindepth 1 -exec rm -rf {} \;
```

Comment:
for information - find have `-delete` option.
```
-delete
             Delete found files and/or directories.  Always returns true.  This executes from the current working directory as find recurses down the tree.  It will not attempt to
             delete a filename with a “/” character in its pathname relative to “.” for security reasons.  Depth-first traversal processing is implied by this option.  The -delete
             primary will fail to delete a directory if it is not empty.  Following symlinks is incompatible with this option.
```
 I'm not sure if it applies in this case, so only for information. Your option is also good.